test-cases/longevity/longevity-large-collections-1h.yaml

```diff
@@ -0,0 +1,25 @@
+test_duration: 60
+stress_cmd: [
+  "JVM_OPTION='-Xms8033M -Xmx8033M -Xmn400M' cassandra-stress user profile=/tmp/large_collections.yaml ops'(insert=3,read1=1,update1=1)' cl=QUORUM duration=20m -mode cql3 native -rate threads=8"
```

Comment:
We have now "custom duration".
Set the duration by default to 12h.

Reply:
If we don't need the 1h test as we have custom duration I should totally remove this file.
We have another (old)file with 12h by default.
This file is also in this commit with added lines.
```
validate_large_collections: true
root_disk_size_runner: 150
```